DevelopGitHub/GitHubExtension/GitHubExtension.Templates/Commands/ITemplatesCommand.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Net.Http;
+using System.Threading.Tasks;
+using GitHubExtension.Templates.CommunicationModels;
+
+namespace GitHubExtension.Templates.Commands
+{
+    public interface ITemplatesCommand
+    {
+        Task<HttpResponseMessage> CreateTemplateAsync(CreateUpdateTemplateModel model);
```

Comment:
Please remove from interface HttpResponseMessage. It's a technical complexity. It's not related to business logic.